releases.go

```diff
@@ -0,0 +1,37 @@
+package versionbundle
+
+import "sort"
+
+// CanonicalizeReleases canonicalizes representation of multiple releases. It
+// makes sure there are no duplicate versions.
+func CanonicalizeReleases(releases []Release) []Release {
+	releases = deduplicateReleases(releases)
+	// TODO: Add changelog deduplication here.
```

Comment:
Changelog deduplication / aggregation / filtering is next task and currently in progress.